code/modules/mob/living/carbon/human/human.dm

```diff
@@ -85,6 +85,8 @@
 			if(mind.changeling)
 				stat("Chemical Storage", "[mind.changeling.chem_charges]/[mind.changeling.chem_storage]")
 				stat("Absorbed DNA", mind.changeling.absorbedcount)
+			if(mind.umbrage_psionics)

```

Comment:
I never had to use 'em. I tried once but ended up crying from frustration and giving up.